src/space.cpp

```diff
@@ -2223,6 +2223,13 @@ static iarf_e do_space(chunk_t *first, chunk_t *second, int &min_sp)
       return(options::sp_paren_qualifier());
    }
 
+   if (  chunk_is_token(first, CT_FPAREN_CLOSE)
```

Comment:
The compiler will hopefully do this anyway, but it *might* not be a bad idea to refactor this and the previous as:
```
if (first is FPAREN_CLOSE)
  if (second is QUALIFIER)
    ...
  else if (second if NOEXCEPT)
    ...
  fi
fi
```

...up to you, though.